### PR TITLE
Update installer script

### PR DESCRIPTION
**Pull request by bovirus**
@hluk 

More use of varables instead fixed text (ex. "CopyQ")
Use of Modern style
Disable "Usepreviouslanguage" setting to enable if new language added in new version will be detected.
Separated setting group with newline for better clearance

Note: for Copyright message could be ok for you
- to remove "(c)" not necessary because Copyright is field name
- to add at the end a starting copyright year/ending copyright year=current year? 
Which is for you your starting copyright year?

### CODE REVIEW

File: shared/copyq.iss [hunk 1]
```diff
@@ -1,38 +1,49 @@
-﻿; 1. Open this file with Inno Setup with Unicode support and preprocessor.
+; 1. Open this file with Inno Setup with Unicode support and preprocessor.
 ; 2. Change "#defines" below (or see below how to use COPYQ_INNO_SETUP environment variable).
 ; 3. Compile "setup.exe".
 
 ; Path for output installation file
-#define Output "."
+#define Output         "."
+#define MyAppName      "CopyQ"
+#define MyAppNameMin   "copy"
```
Comment (hluk) [inline shared/copyq.iss]:
```suggestion
#define MyAppNameMin   "copyq"
```

**Review by hluk — COMMENTED** [review]
Thanks for the fixes!

I believe the starting year is 2009.

File: shared/copyq.iss (hunk 1 above)
Comment (bovirus) [reply shared/copyq.iss]: Tnahkls for the report about ypo error

Comment (bovirus) [reply shared/copyq.iss]: I updated the installer script with "MyAppCopyrightStartYear" and "MyAppCopyrighTEndYear variables".

Please check it. Thanks.

**Comment by hluk** [comment]
Merged, thanks!

**Pull request merged**